weaviate/rbac/role_all_getter.go

```diff
@@ -0,0 +1,27 @@
+package rbac
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/connection"
+	"github.com/weaviate/weaviate-go-client/v4/weaviate/except"
+	"github.com/weaviate/weaviate/entities/models"
+)
+
+type RoleAllGetter struct {
+	connection *connection.Connection
+}
+
+func (rag *RoleAllGetter) Do(ctx context.Context) ([]*models.Role, error) {
```

Comment:
not the biggest fan of using models.Role as return because of the *string in there